src/ngx-json-viewer/ngx-json-viewer.component.ts

```diff
@@ -101,6 +116,12 @@ export class NgxJsonViewerComponent implements OnChanges {
     return segment;
   }
 
+  private isLink(value: string): boolean {
+    return value.substring(0, 8) === 'https://' ||
+      value.substring(0, 7) === 'http://' ||
+      value[0] === '/'
```

Comment:
What if this is changed to a RegExp, the scheme is not case sensitive, and is also posible `http:(domain)` 
[Reference](https://learn.microsoft.com/en-us/deployedge/edge-learnmmore-url-list-filter%20format)

Reply:
Sorry, I was delayed a bit. I pushed and would appreciate your review.